tcms/telemetry/static/telemetry/js/testing/utils.js

```diff
@@ -22,3 +22,10 @@ function loadInitialProduct(callback = () => {}) {
 function loadInitialTestPlans() {
     jsonRPC('TestPlan.filter', {}, data => updateSelect(data, '#id_test_plan', 'plan_id', 'name'));
 }
+
+function showOnlyRoundNumbers(number) {
+    if (number % 1 === 0) {
+        return number;
+    }
+    return '';
```

Comment:
I'm not exactly sure what this function does but if the number isn't round (isn't integer) then we show empty string. Can't we instead round the numbers unconditionally ?

Reply:
the idea of this function is to render the scale label on the y axis. 
why do we need this: because if you have a smal amount of test executions, let's say 3, your 3 test executions would span over the whole screen. the y axis will have multiple labels - 1, 1.1, 1.2, 1.3.. all the way to 3. that of course is useless, because you can't have 1.1 test execution. if we just round the number we will have labels - 1, 1, 1, 1, 1, 1, 2, 2, 2, which is ugly. so we just show the whole numbers - 1, 2, 3.

pictures:
how it is now:
<img width="1648" alt="Screenshot 2019-07-11 at 15 23 39" src="https://user-images.githubusercontent.com/18421997/61050554-ef5ef600-a3ef-11e9-91ee-68259a7e67ef.png">
how it is without this function:
<img width="1655" alt="Screenshot 2019-07-11 at 15 24 46" src="https://user-images.githubusercontent.com/18421997/61050595-0d2c5b00-a3f0-11e9-9ef3-f590a1104acd.png">
how it is if we just round the number:
<img width="1649" alt="Screenshot 2019-07-11 at 15 25 34" src="https://user-images.githubusercontent.com/18421997/61050638-292ffc80-a3f0-11e9-8dae-d14818286f9d.png">